CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ### 2.2.1 (Next)
 
+* [#90](https://github.com/acm19/aws-request-signing-apache-interceptor/pull/90): Add support for Serverless OpeanSearch - [@acm19](https://github.com/acm19).
```

Comment:
Typo: OpeanSearch -> OpenSearch

We use the terminology of "OpenSearch Serverless" and "Managed OpenSearch".

Reply:
Fixed, thanks.